packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -492,7 +492,7 @@ class _BottomNavigationTile extends StatelessWidget {
         message: item.label!,
         preferBelow: false,
         verticalOffset: selectedIconSize + selectedFontSize,
-        child: result,
+        child: ExcludeSemantics(child: result),
```

Comment:
instead of excluding the the result entirely, we should set excludeFromSemantics to true for tooltip

Reply:
I have added tests as well as did the changes.